frontend/src/components/Cases/AccordionSvg.jsx

```diff
@@ -4,43 +4,14 @@ import {
   XCircleFill,
 } from "react-bootstrap-icons";
 
-const AccordionSvg = ({ testResult }) => {
-  let svgComponent;
-
-  switch (testResult) {
-    case "skipped":
-      svgComponent = <ExclamationOctagon />;
-      break;
-    case "errored":
-      svgComponent = <ExclamationOctagon />;
-      break;
-    case "successful":
-      svgComponent = <CheckCircleFill />;
-      break;
-    case "unexpectedlyValid":
-      svgComponent = <CheckCircleFill />;
-      break;
-    case "unexpectedlyInvalid":
-      svgComponent = <XCircleFill />;
-      break;
-    default:
-      return null;
+const AccordionSvg = ({ result }) => {
+  if (result.state === "skipped" || result.state === "errored") {
+    return <td className='text-center text-bg-warning'><ExclamationOctagon /></td>
```

Comment:
```html
return <td className=`text-center ${result.state === "skipped" ? "text-bg-warning" : "text-bg-danger"}`><ExclamationOctagon /></td>
```

Reply:
Sorry, I didn't see you last comments. I believe the background for skipped and errored cases were the same? But still, I agree with you that it should look like differently, so changed it. Added also a title to provide even more information (when I was using the site I felt like I don't really know what these colors and icons mean :))